src/lib_ccx/ccx_encoders_common.c

```diff
@@ -1089,14 +1089,9 @@ static int init_output_ctx(struct encoder_ctx *ctx, struct encoder_cfg *cfg)
 				ret = init_write(&ctx->out[1], create_outfilename(basefilename, "_2", extension), cfg->with_semaphore);
 				check_ret(ctx->out[1].filename);
 			}
-			else if (cfg->extract == 1)
-			{
-				ret = init_write(ctx->out, strdup(cfg->output_filename), cfg->with_semaphore);
-				check_ret(cfg->output_filename);
-			}
```

Comment:
These statements are redundant according to me, as the below else condition does the same job.